src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -0,0 +1,101 @@
+.card {
+  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary */
```

Comment:
Can you link to the spec on what the border color change will mean?

Reply:
Like the figma file? sure!

Reply:
If the Figma file explains what the meaning of the color is, that'd be great to look at!